webapp/portlet/src/main/webapp/vue-apps/activity-stream/components/ActivityStream.vue

```diff
@@ -98,11 +97,13 @@ export default {
     },
     refreshActivityActions() {
       const extensions = extensionRegistry.loadExtensions(this.extensionApp, this.activityActionExtension);
+      const newActivityActions=new Array(extensions.length);
       extensions.forEach(extension => {
         if (extension.id) {
-          this.activityActions[extension.id] = extension;
+          newActivityActions[extension.id] = extension;
```

Comment:
You changed an object into an array, please make sure to preserve the same data type